utils/test_helper.ts

```diff
@@ -185,4 +187,44 @@ export class TestHelper {
         };
         return Object.assign({}, defaultGroup, override);
     }
+
+    public static getPostMock(override: Partial<Post>): Post {
+        const defaultPost: Post = {
+            id: 'post_id',
+            create_at: 1,
+            update_at: 1,
+            edit_at: 1,
+            delete_at: 1,
+            is_pinned: false,
+            user_id: 'user_id',
+            channel_id: 'channel_id',
+            root_id: 'root_id',
+            parent_id: 'parent_id',
+            original_id: 'original_id',
+            message: 'message',
+            type: 'system_add_remove',
+            props: {props: 'props'},
+            hashtags: 'hashtags',
+            pending_post_id: 'pending_post_id',
+            reply_count: 1,
+            metadata: {
+                embeds: [],
+                emojis: [],
+                files: [],
+                images: {'image-1': {height: 30, width: 30}},
+                reactions: [],
+            },
+        };
+        return Object.assign({}, defaultPost, override);
+    }
```

Comment:
This is great! But I just reviewed a PR from @abdusabri that has a similar addition: https://github.com/mattermost/mattermost-webapp/pull/6717 As soon as we merge one of the PRs, the other one will need to fix the conflict that will arise here!

Reply:
Thanks for the mention @agarciamontoro! I think we should try to keep specific keys like `images` out of the mock utility to avoid the potential need for updating multiple snapshots and/or the need to keep passing the `override`  argument when using the mock utility (not all posts has images, hashtags, replies). I've also seen the common approach in the repo for numeric defaults to be `0`.

Anyway, happy to update my PR to whatever is here if it gets merged before mine 🙂 

And thanks for contributing @Tak-Iwamoto! 🙂

Reply:
I think that's a good suggestion, keeping this to the bare default, and adding what we need (the image, in this case) in the caller code.

Reply:
https://github.com/mattermost/mattermost-webapp/pull/6717 was just merged. @Tak-Iwamoto, could you merge master? You'll get a conflict in this file. I suggest to keep the changes from master and adapt your code to call `getPostMock` with the images you need. Don't hesitate to ask if you have any problems resolving the merge conflict!

Reply:
Thanks @agarciamontoro  and @abdusabri !